src/main/java/io/github/moulberry/notenoughupdates/commands/Commands.java

```diff
@@ -336,6 +337,14 @@ public void processCommand(ICommandSender sender, String[] args) {
         }
     });
 
+    SimpleCommand neuRepoMode = new SimpleCommand("neurepomode", new SimpleCommand.ProcessCommandRunnable() {
```

Comment:
why is this necessary? I feel like this will just confuse people more than it will help

Reply:
Its useful and been asked before